src/app/PCM/templates/tpl_vertical_cards/index.js

```diff
@@ -0,0 +1,139 @@
+import { Box, Grid, HStack, VStack } from '@chakra-ui/react'
+import { equals, filter, includes, map } from 'ramda'
+import { getColumnDefs, getFields } from '../../helpers/sbe-utils'
+import { selectCode, selectKeys } from 'redux/db/selectors'
+
+import Attribute from 'app/BE/attribute'
+import ContextMenu from 'app/BE/context'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import Title from 'app/SBE/table/Title'
+import { faEllipsisV } from '@fortawesome/free-solid-svg-icons'
+import { useGetActionsFromCode } from 'app/SBE/utils/get-actions'
+import useGetMappedBaseEntity from 'app/PCM/helpers/use-get-mapped-base-entity'
+import useProductColors from 'utils/productColors'
+import { useSelector } from 'react-redux'
+
+const TemplateVerticalCards = ({ mappedPcm, depth }) => {
+  const sbeCodePrefix = mappedPcm?.PRI_LOC1 || ''
+  const keys = useSelector(selectKeys)
+  const sbeCode = (filter(key => includes(sbeCodePrefix)(key) && !includes('@')(key))(keys) || [
+    '',
+  ])[0]
+  const actions = useGetActionsFromCode(sbeCode) || []
+  const actionCodes = map(act => act?.attributeCode || '')(actions)
+  const mappedSbe = useGetMappedBaseEntity(sbeCode)
+  const mappedValues = getFields(getColumnDefs(mappedSbe))
+
+  const rows = useSelector(selectCode(sbeCode, 'rows')) || []
+  const { buttonBackgroundColor } = useProductColors()
+
+  return (
+    <>
+      {!!rows.length && (
+        <Box padding={'10px'}>
+          <Box paddingBottom={3}>
+            <Title sbeCode={sbeCode} />
+          </Box>
+
+          <Grid gap={4}>
+            {rows.map(item => (
+              <Card
+                actions={actionCodes}
+                key={`CARD-${item || ''}`}
+                mappedValues={mappedValues}
+                baseEntityCode={item}
+                primaryColor={buttonBackgroundColor}
+                sbeCode={sbeCode}
+              />
+            ))}
+          </Grid>
+        </Box>
+      )}
+    </>
+  )
+}
+
+//Image is hardcoded for the demo, need to remove it after the demo.
```

Comment:
remove this line if the image is not hardcoded anymore.

Reply:
done